cmd/root.go

```diff
@@ -177,20 +200,32 @@ func Execute() {
 				fields["hint"] = e.Hint
 			}
 		}
+
 		logger.WithFields(fields).Error(err)
+		if c.loggerIsRemote {
+			fallbackLogger.WithFields(fields).Error(err)
+			cancel()
+			<-c.loggerStopped // TODO have a timeout?
```

Comment:
Not sure if we need a timeout if `^C` can stop this, no?

Reply:
I guess not, and also the question is ... how big of timeout? I don't want to add one more configuration so :shrug: 
cc @na-- opinion ?

Reply:
I think this timeout should be handled by each logger individually, so it's easier to configure. And, unless I'm mistaken, the loki logger sort of has that in https://github.com/loadimpact/k6/blob/5091636dbf920aadb51d52d53f7d43fd30049dd6/log/loki.go#L97

That said, it won't hurt to add another hardcoded 5 second max timeout here, with a warning message to the fallback logger, just in case...

Reply:
Okay, I am gonna add global timeout